### DISCUSSION TITLE
How to use reverse api?

### DISCUSSION THREAD

**Discussion by maxingg** [mediagis/nominatim-docker#368] [Q&A]
I' ve started nominatim by docker, but it fails when I request the following request. 
```python
https://localhost:8080/reverse?lat=-37.812337971404176&lon=144.97309708569782&extratags=1
```

**Comment by mtmail** [comment]
"It fails" is not a good description and people would have to guess what you mean.

Does your server have https:// configured? If not try http:// protocol.

**Comment by maxingg** [comment]
I tried to change https:// to http://
```shell
http://localhost:8080/reverse?format=jsonv2&lat=-37.812337971404176&lon=144.97309708569782&extratags=1
```
But the html returns
```json
{
"error": "Unable to geocode"
}
```

**Comment by mtmail** [comment]
"Unable to geocode" is the response when Nominatim doesn't find anything on or near the coordinates. Have you imported data for Melbourne, Australia? Did you see any errors during the data import?

**Comment by acpguedes** [comment]
In my case the reverse might not work since TIGER hasn't the Brazil's postcodes.

But I have a shapefile mapping the lat-long to ZIP code.

I wonder if there is a way to use this data
